packages/react/src/components/Carousel/Carousel.tsx

```diff
@@ -390,6 +393,7 @@ class Carousel extends AutoControlledComponent<WithAsProp<CarouselProps>, Carous
       })
     ) : (
       <Text
+        aria-hidden="true"
```

Comment:
I don't understand. If we are hiding this then what is the point in having it in the first place?

Reply:
There was relevant complain that with virtual cursor navigation user hit the text twice. 
I am hiding here the second text. First text appearance still stay in each tab/slide.